src/main/java/com/cinema/model/ShoppingCart.java

```diff
@@ -0,0 +1,76 @@
+package com.cinema.model;
+
+import java.util.List;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_cart")
+public class ShoppingCart {
```

Comment:
Why is this in this PR?

Reply:
Don't know - may be because I have not merge previous HW?